src/domain/hanke/edit/Form2.tsx

```diff
@@ -55,7 +55,7 @@ const Form2: React.FC<FormProps> = ({ control, formData, errors, register }) =>
 
   return (
     <div className="form2">
-      <Text tag="h2" spacing="s" weight="bold">
+      <Text tag="h2" spacing="s" weight="bold" data-testid="hankeFormHeader">
```

Comment:
hankeFormHeader asetettu form2 ja form3:lle mutta näitä ei nähtävästi käytetty testikoodissa validaatiossa muuten kuin fillForm0:ssa ja fillForm4:ssa?